src/app.md

```diff
@@ -725,6 +726,50 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-semver-latest]: #ex-semver-latest
+<a name="ex-semver-latest"></a>
+## Find the latest version satisfying given range
+[![semver-badge]][semver] [![cat-config-badge]][cat-config]
+
+Given an array of unparsed version `&str`s, iterate and filter against a semver `VersionReq` and find the latest `Version`.
+
+```rust
+#[macro_use]
+extern crate error_chain;
+extern crate semver;
+
+use semver::Version;
+use semver::VersionReq;
+
+mod errors {
```

Comment:
You can make your example more concise by removing `mod errors` scope.

Reply:
this whole mod can be deleted.